Brain/AbstractBrain.h

```diff
@@ -105,7 +105,6 @@ class AbstractBrain {
          << std::endl
          << "Exiting." << std::endl;
     exit(1);
-    return makeCopy();
```

Comment:
Code cleanup would be easier to review if it were in a different PR.

Reply:
Agreed. In this case, can I leave it in? This is a really obvious one.